[RHACS] Modularized and removed duplicate docs, updated CLI options (#87677)
openshift/openshift-docs: modules/roxctl-netpol-generate.adoc

|===
|Option |Description

|`--dnsport <int_or_string>`

> a|Specify the DNS port or a named port that you want to use in the egress rules of synthesized network policies. For example:
> 
> * `roxctl netpol generate --dnsport 5353 <other_options>`.
> * `roxctl netpol generate --dnsport foo-dns <other_options>`.

Reviewer: ```suggestion
a|Specify the DNS port or a named port that you want to use in the egress rules of synthesized network policies. For example:

* `roxctl netpol generate --dnsport 5353 <other_options>`
* `roxctl netpol generate --dnsport foo-dns <other_options>`.
```

|`--strict`
|Treat warnings as errors. The default value is `false`.
|===

[NOTE]
====

> By default, the `roxctl netpol generate` command uses port `53` for DNS connections.

Reviewer: ```suggestion
If you do not specify a port, the `roxctl netpol generate` command uses port `53` for DNS connections.
```

Reviewer: https://www.ibm.com/docs/en/ibm-style?topic=elements-command-syntax#:~:text=Use%20the%20set%20monitor%20command%20to%20turn%20the%20monitor%20on%20or%20off.%20If%20you%20do%20not%20specify%20on%20or%20off%2C%20off%20is%20selected%20automatically.

> If you are using {ocp}, you might need to change the port when generating network policies with `roxctl` because, by default,  {ocp} uses port `5353` and assigns the name `dns` for this port. You can use the `--dnsport` option to override the default DNS port. For example:

Reviewer: ```suggestion
If you are using {ocp}, you might need to change the port when generating network policies by using the `roxctl` CLI. If you do not change the port,  {ocp} uses port `5353` and assigns the name `dns` for this port automatially. You can use the `--dnsport` option to override the default DNS port. For example:
```

Reviewer: https://www.ibm.com/docs/en/ibm-style?topic=elements-command-syntax#:~:text=Use%20the%20set%20monitor%20command%20to%20turn%20the%20monitor%20on%20or%20off.%20If%20you%20do%20not%20specify%20on%20or%20off%2C%20off%20is%20selected%20automatically.

> * `roxctl netpol generate --dnsport 5353 <other_options>`, or
> * `roxctl netpol generate --dnsport dns <other-options>`.

Reviewer: ```suggestion
* `roxctl netpol generate --dnsport 5353 <other_options>`
* `roxctl netpol generate --dnsport dns <other-options>`.
```
